pandas/tests/arrays/test_datetimelike.py

```diff
@@ -1,13 +1,51 @@
 # -*- coding: utf-8 -*-
 import numpy as np
+import pytest
 
 import pandas as pd
+import pandas.util.testing as tm
 
 from pandas.core.arrays.datetimes import DatetimeArrayMixin
 from pandas.core.arrays.timedeltas import TimedeltaArrayMixin
 from pandas.core.arrays.period import PeriodArrayMixin
 
 
+# TODO: more freq variants
+@pytest.fixture(params=['D', 'B', 'W', 'M', 'Q', 'Y'])
+def per_index(request):
```

Comment:
The name confused me at first, I read "per" as in "miles per hour". Maybe just `period_index`, and `datetime_index`?

Reply:
Good idea, will update.